### DISCUSSION TITLE
Question about documentation: FE Compatibility Test, first step is starting the FE by 'start_fe.sh --daemon'

### DISCUSSION THREAD

**Discussion by patrickdung2022** [apache/doris#32894] [A - General / Q&A]
https://doris.apache.org/docs/admin-manual/cluster-management/upgrade/

After the pre-upgrade work, https://doris.apache.org/docs/admin-manual/cluster-management/upgrade/#upgrade-pre-work, in the 'FE Compatibility Test'

The first step is 
Use the new version alone to deploy a test FE process
```
sh ${DORIS_NEW_HOME}/bin/start_fe.sh --daemon
```

At this point, I think the FE is started using the default configuration.
Then in next steps, it mention updating configuration or copy meta data.

2. Modify the FE configuration file fe.conf for testing
```
vi ${DORIS_NEW_HOME}/conf/fe.conf
```
....
3. Copy the metadata directory doris-meta of the online environment Master FE to the test environment

So, the first step by staring the FE daemon(start_fe.sh --daemon), it does not seem to make sense.
What do you think?

**Comment by yagagagaga** [comment]
It makes sense because it is actually a cluster initialization action

**Reply by patrickdung2022** [reply]
'actually a cluster initialization action'

What I don't understand is:

It started the new testing Doris without stopping it and then edit configuration and copy metadata from production.
Also, without mentioning about stopping the new testing Doris between step 1 and step5, on step 6, it mentioned starting the testing Doris again

```
6. In the test environment, run the startup FE

If the version is greater than or equal to 2.0.2, run the following command

sh ${DORIS_NEW_HOME}/bin/start_fe.sh --daemon --metadata_failure_recovery

If the version is less than 2.0.2, run the following command

sh ${DORIS_NEW_HOME}/bin/start_fe.sh --daemon
```
